### PR TITLE
feat(*): add scripts/jobs for signing helm charts

### PR DESCRIPTION
**Pull request by vdice**
ref https://github.com/deis/workflow/issues/595

TODO
- [x] test affected pipelines 
  * workflow: https://ci.deis.io/job/workflow-chart-publish/
  * component: https://ci.deis.io/job/router-chart-publish/ (just one component needed)

### CODE REVIEW

File: jobs/component_chart_publish.groovy [hunk 1]
```diff
@@ -59,45 +59,46 @@ repos.each { Map repo ->
...
+            #!/usr/bin/env bash
+
+            set -eo pipefail
+            set -x
+
+            # check out RELEASE_TAG tag
+            commit="\$(git checkout "\${RELEASE_TAG}" && git rev-parse HEAD)"
+
+            if [ -d charts ]; then
+              cd charts
```
Comment (bacongobbler) [inline jobs/component_chart_publish.groovy:73]: do we want to `pushd/popd` from this directory so the next script (if we include another at some point) is running from the original directory, or is this fine the way it is?

Comment (vdice) [reply jobs/component_chart_publish.groovy:73]: @bacongobbler I actually haven't used those utilities before, but if it cleans up and/or makes these script(s) less error-prone, I'd definitely endorse it.

**Comment by vdice** [comment]
realize I also need to hook up the existing component/workflow chart pipelines to this new `helm-chart-sign` job... additions coming soon

**Comment by vdice** [comment]
additions mentioned in https://github.com/deis/jenkins-jobs/pull/282#issuecomment-258993169 added; next need to test affected jobs by building them off this branch... **edit: done**

File: jobs/component_chart_publish.groovy (hunk 1 above)
Comment (mboersma) [reply jobs/component_chart_publish.groovy:73]: It doesn't look like this script does a `cd ..` later, so probably `pushd/popd` aren't necessary here.
https://en.wikipedia.org/wiki/Pushd_and_popd

**Pull request merged**